ansible_collections/arista/avd/molecule/eos_validate_state/reports/FABRIC-state.csv

```diff
@@ -1,1711 +1,1711 @@
+34,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: Loopback0 (IP: 10.255.0.3) - Destination: dc2-leaf2b Loopback0 (IP: 10.255.128.16),NOT RUN,
+35,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: Loopback0 (IP: 10.255.0.3) - Destination: dc2-leaf3a.arista.com Loopback0 (IP: 10.255.128.17),NOT RUN,
+36,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: Loopback0 (IP: 10.255.0.3) - Destination: dc2-leaf3b.arista.com Loopback0 (IP: 10.255.128.18),NOT RUN,
+37,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: Loopback0 (IP: 10.255.0.3) - Destination: dc2-spine1 Loopback0 (IP: 10.255.128.11),NOT RUN,
+38,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: Loopback0 (IP: 10.255.0.3) - Destination: dc2-spine2 Loopback0 (IP: 10.255.128.12),NOT RUN,
+39,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: P2P Interface Ethernet1 (IP: 10.255.255.1) - Destination: dc1-spine1 Ethernet1 (IP: 10.255.255.0),NOT RUN,
+40,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: P2P Interface Ethernet2 (IP: 10.255.255.3) - Destination: dc1-spine2 Ethernet1 (IP: 10.255.255.2),NOT RUN,
+41,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: P2P Interface Ethernet6 (IP: 10.255.255.0) - Destination: dc1-wan1 Ethernet1 (IP: 10.255.255.1),NOT RUN,
+42,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: P2P Interface Ethernet7 (IP: 10.255.255.4) - Destination: dc1-wan2 Ethernet1 (IP: 10.255.255.5),NOT RUN,
+43,dc1-leaf1a,Field Notices;Software,VerifyFieldNotice44Resolution,Verifies the device is using an Aboot version that fix the bug discussed in the field notice 44 (Aboot manages system settings prior to EOS initialization),,NOT RUN,
+44,dc1-leaf1a,Field Notices;Software,VerifyFieldNotice72Resolution,"Verifies if the device has exposeure to FN72, and if the issue has been mitigated",,NOT RUN,
+45,dc1-leaf1a,Hardware,VerifyAdverseDrops,Verifies there are no adverse drops on DCS7280E and DCS7500E,,NOT RUN,
+46,dc1-leaf1a,Hardware,VerifyEnvironmentCooling,Verifies the status of power supply fans and all fan trays.,,NOT RUN,
+47,dc1-leaf1a,Hardware,VerifyEnvironmentCooling,Verifies the status of power supply fans and all fan trays.,Accepted States: 'ok',NOT RUN,
+48,dc1-leaf1a,Hardware,VerifyEnvironmentPower,Verifies the power supplies status.,,NOT RUN,
+49,dc1-leaf1a,Hardware,VerifyEnvironmentPower,Verifies the power supplies status.,Accepted States: 'ok',NOT RUN,
```

Comment:
Why testcase running two times on single device, also for first time there is no accepted states but its there for second one.

Reply:
In molecule, we provide a custom catalog with random tests: https://avd.sh/en/latest/roles/eos_validate_state/ANTA-Preview.html. Check the custom catalog section.

The first test is from the custom catalog and the second is generated from AVD so it has a  custom field to give the inputs/context of the test.